### PR TITLE
docs: fix some typo

### PR DESCRIPTION
**Pull request by nozo-moto**
## Description

fix some typo

## Checklist
- [ ] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [ ] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: docs/tutorials/kubernetes/kyverno.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ spec:
 
 ### Apply the policy to your Kubernetes cluster
 
-Ensure that you have Kyverno already deployed and running on your cluster -- for instance throught he Kyverno Helm Chart.
+Ensure that you have Kyverno already deployed and running on your cluster -- for instance thought he Kyverno Helm Chart.
```
Comment (knqyf263) [inline docs/tutorials/kubernetes/kyverno.md:53]: I think it should be `through`

**Review by knqyf263 — COMMENTED** [review]
Thanks for your contribution.

File: docs/tutorials/kubernetes/kyverno.md (hunk 1 above)
Comment (nozo-moto) [reply docs/tutorials/kubernetes/kyverno.md]: Thanks I fixed it 👍 
https://github.com/aquasecurity/trivy/pull/3171/commits/f146861a79d4bc065af65e3c13d1101eb631dfad

**Comment by nozo-moto** [comment]
@knqyf263 could you retry `Test / Build Test (pull_request)` test ?
it seems failed by apk error.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**